Tests/scripts/prepare_private_content_packs_for_testing.sh

```diff
@@ -5,7 +5,6 @@ set -e
 
 GIT_BRANCH=${BRANCH_NAME:-unknown}
 GITHUB_RUN_NUMBER=${GITHUB_RUN_NUMBER:-00000}
-ARTIFACTS_FOLDER=${ARTIFACTS_FOLDER}
```

Comment:
Why deleting this, instead of setting a default value similar to what is done for the first two env vars?